### PR TITLE
Added sound commands and fixed two bugs here

### LINKED ISSUE #37: Print list of current sounds and remove existing sounds
There is a need to orient what sounds are in the bot already uploaded and what names are taken to not generate more  errors.

### LINKED ISSUE #41: Create docker image on official Alpine image
### Current
Currently our docker image is built based on an unofficial Alpine image with FFmpeg library with python 3.8.

### Task
Prepare Dockerfile based on official Alpine Linux image. It requires at least additional installation of FFmpeg and maybe python with pip package installer.

### Links
Link to official docker image: [DockerHub](https://hub.docker.com/_/alpine)

### PR DESCRIPTION
**Pull request by Niewidzialny84**
This will fix the never ending `settimezone` command in #36 . Also making sure that requirements.txt contains all the necessary packages fixing #35 . 
Added the `soundlist` command to get the current list of sounds
Added the `soundrm <sound>` command to remove given sound
Changed the `us <sound>` to `soundupload`
Moved sounds element to separare file
Changed `sound` messages resolve #37 
Resolve #41  using python alpine image, also added a README.md file with how to install

### CODE REVIEW

**Comment by Niewidzialny84** [comment]
Fixed #41 using python alpine image, also added a README.md file with how to install

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+### How to build docker image
+```
+docker build -t seker212/discord_bot:latest .
+```
+
+## Running normally
+### How to install
+1. Clone repository from github
+```
+git clone https://github.com/seker212/Discord_bot.git
```
Comment (seker212) [inline README.md:30]: Rly?!?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+```bash
+python3 -m venv env
+#Linux version
+source rest/bin/activate
+#Windows version
+env\Scripts\activate
+```
+3. Install neccesary packages
+```
+pip install requirements.txt
```
Comment (seker212) [inline README.md:42]:
```suggestion
pip install -r requirements.txt
```

**Review by seker212 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Niewidzialny84) [reply README.md:30]: Yes

File: README.md (hunk 2 above)
Comment (Niewidzialny84) [reply README.md]: the `python -m` was also missing there

**Review by seker212 — APPROVED** [review]

**Pull request merged**